rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -0,0 +1,179 @@
+= Manage Credentials
+
+Credentials store login data for external applications or web services. By using a credential, a bot can log in to an application during process runs. Passwords are encrypted. Only affiliated users can view, edit, and use the credentials.
+
+You can create the following types of credentials:
+
+* *AWS Credentials*
++
+For accessing Amazon Web Services to use them with xref:rpa-builder::toolbox-aws-document-processing.adoc[RPA Builder - Document Processing with AWS]
+* *OAuth Credentials*
++
+For accessing the Microsoft Outlook email application via Microsoft Azure with xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)]
```

Comment:
```suggestion
For accessing the Microsoft Outlook email application via Microsoft Azure with xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)]:
```

Reply:
changed all three, but with full stop instead of colon at the end

Reply:
Oh yeah, I see now. (Sorry about the colons :)